README.md

```diff
@@ -1,71 +1,78 @@
-# Open LossLess in Standards
+# Open LossLess in STanDards
+
+An open effort to standardize open formats.
 
 ## Foreword
 
-In cases where digital loss is not acceptable, lossless data compression has been changing the way the world stores and transmits its data. Nowadays most digital office documents use lossless compression (.docx, .xlsx, .odt, .pages) and lossless images are more ubiquitous (.png). Although uncompressed digital storage is an option, uncompressed data generally contains no fixity mechanisms and requires a generation of e-waste at a much faster rate than lossless formats. When the preservation of quality is a requirement, lossless compression is the green option.
+In cases where digital loss is not acceptable, lossless data compression has been changing the way the world stores and transmits its data. Nowadays most digital office documents use lossless compression (.docx, .xlsx, .odt, .pages) and lossless images are more ubiquitous (.png). Although uncompressed digital storage is an option, uncompressed data generally contains no fixity mechanisms and generates e-waste at a much faster rate than lossless formats. When the preservation of quality is a requirement, lossless compression is the green option.
 
-Although lossless formats have been well adopted for images, documents and other media types; the adoption of lossless formats for audiovisual recordings has been much slower. Given the opportunity for lossless compression in audiovisual media, we also propose a focus on open formats, meaning:
+Although lossless formats have been well adopted for images, documents and other media types, the adoption of lossless formats for audiovisual recordings has been much slower. Given the opportunity for lossless compression in audiovisual media, we also propose a focus on open formats, meaning:
 
-- freedom to read, write, and use the format (NO PATENT)
-- freedom to read, understand, and share the format's specification (NO PAYWALL)
-- freedom to modify and distribute the format
+- NO PATENT -- freedom to read, write, and use the format
+- NO PAYWALL -- freedom to read, understand, and share the format's specification
+- NO PROHIBITIONS -- freedom to modify and distribute the format
 
 We propose accelerating the implementation of lossless audiovisual formats with the following coordinated efforts:
 
-- coordinated registration efforts for lossless formats with the principle audiovisual standards organizations (at least IANA, IETF, ISO, SMPTE)
+- coordinated registration efforts for lossless formats with the principle audiovisual standards organizations (IANA, IETF, ISO, SMPTE, et al)
```

Comment:
I would prefer to omit "et al" or "etc". The first version (with "at least") gives the charter a limit whereas the "et al" version is open-ended. I would prefer the constraint of the list of four. If another is added I would prefer to update this doc rather than consider it included in "etc".